sdk/maps/azure-maps-render/MANIFEST.in

```diff
@@ -0,0 +1,7 @@
+recursive-include tests *.py *.yaml
+recursive-include samples *.py *.md
+include *.md
+include LICENSE
```

Comment:
Actually we don't have the `LICENSE` file. You can copy from other modules to search and render :)